pkg/kaadm/kaadm.go

```diff
@@ -0,0 +1,55 @@
+package kaadm
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/karmada-io/karmada/pkg/kaadm/kubernetes"
+	"github.com/karmada-io/karmada/pkg/kaadm/options"
+)
+
+// NewCmdInit init karmada.
+func NewCmdInit() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "init",
+		Short: "bootstrap install karmada (default in kubernetes)",
+		Long:  `Installation options.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			kubernetes.Deploy()
+		},
+		Example: "kubectl karmada init --master=xxx.xxx.xxx.xxx",
```

Comment:
If we can give more examples would be wonderful!
Like [karmada get](https://github.com/karmada-io/karmada/blob/006bccd0bdc992963c1e850fb3d5668311964088/pkg/karmadactl/get.go#L543-L558) or [kubeadm](https://github.com/kubernetes/kubernetes/blob/c83a94da72416219cf9f022dc44b0e6db158f092/cmd/kubeadm/app/cmd/cmd.go#L35-L66).